README.md

```diff
@@ -655,6 +655,33 @@ k3os:
   - "key1=value1:NoExecute"
 ```
 
+### `k3os.etcd`
+
+Available for AMD64 and ARM64
+
+Add the etcd service to the node which k3s can then use as a storage backend.
+
+ETCD_HOST_IP is retrieved by the ip address on the iface and can be used in the etcd args.
+ETCD_TOKEN is set via the k3os.etcd.token
```

Comment:
I can see `ETCD_NAME` in `args` but not here. Maybe add it here or remove it in `args`?

Reply:
@zimme ETCD_NAME is automatically set in /etc/conf.d/etcd-service and by default is the hostname so can be used in the args.

I'll update the documentation to make it clear it's available as a variable in the args.

Reply:
@zimme PR Updated with improved documentation including ETCD_NAME.